Jenkinsfile-komodo

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    agent { label 'si-build' }
+    stages {
+        stage('checkout komodo tag') {
+            steps {
+                sh '''
+source /project/res/komodo/$KOMODO_VERSION/enable
+
+GIT=/prog/sdpsoft/bin/git
+
+# find and check out the code that was used to build libres for this komodo relase
+echo "checkout tag from komodo"
+EV=$(cat /project/res/komodo/$KOMODO_VERSION/$KOMODO_RELEASE | grep "ert:" -A2 | grep "version:")
+EV=($EV)    # split the string "version: vX.X.X"
+EV=${EV[1]} # extract the version
+echo "Using Ert version $EV"
+$GIT checkout $EV
+'''
```

Comment:
Would it be worth moving these lines to a separate shell script as well?

Reply:
That might be good yes. I'll do that when I get the build to work.

Reply:
Did this in both repos